zerver/openapi/zulip.yaml

```diff
@@ -8221,6 +8221,10 @@ paths:
                             "msg": "You don't have permission to delete this message",
                             "result": "error",
                           }
+  /tus:
+    $ref: https://raw.githubusercontent.com/tus/tus-resumable-upload-protocol/449ec910abb3222a05dde230a49cec964bf5aeda/OpenAPI/openapi3.yaml#/paths/~1files
+  /tus/{id}:
+    $ref: https://raw.githubusercontent.com/tus/tus-resumable-upload-protocol/449ec910abb3222a05dde230a49cec964bf5aeda/OpenAPI/openapi3.yaml#/paths/~1files~1{id}
```

Comment:
This makes URLs like `http://localhost:9991/api/FilePatch` exist, but crash. Maybe we should skip for now.

Reply:
The below fixes the first couple causes for crash, but there's a few more. And I don't think the URL pattern is desired either. So probably a whole project to make this work.

```
@@ -338,7 +339,7 @@ NO_EXAMPLE = object()
 
 
 class Parameter(BaseModel):
-    kind: Literal["query", "path", "formData"]
+    kind: Literal["query", "path", "formData", "header"]
     name: str
     description: str
     json_encoded: bool
@@ -375,11 +376,12 @@ def get_openapi_parameters(
         else:
             example = schema.get("example", NO_EXAMPLE)
 
         parameters.append(
             Parameter(
                 kind=parameter["in"],
                 name=parameter["name"],
-                description=parameter["description"],
+                description=parameter.get("description", "No description"),
                 json_encoded=json_encoded,
                 value_schema=schema,
                 example=example,
```